Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py

```diff
@@ -131,6 +131,60 @@ def create_entry(title: str, data: dict, ec: dict):
     }
 
 
+def read_zip_to_bytes(filename):
+    """
+  Reads the entire zip file into a bytes object in chunks.
+
+  Args:
+      filename: Path to the zip file.
+
+  Returns:
+      A bytes object containing the complete zip file content.
+  """
+    with open(filename, "rb") as zip_file:
+        data = b''
+        for chunk in iter(lambda: zip_file.read(1024), b''):
+            data += chunk
+    return data
+
+
+def prepare_create_function_kwargs(args: dict[str, str]):
+    """
+    Prepare arguments to be sent to the API
+    """
+    create_function_api_keys = ['FunctionName', 'Runtime', 'Role', 'Handler', 'Description', 'PackageType']
+    kwargs = {}
+
+    if code_path := args.get('code'):
+        file_path = demisto.getFilePath(code_path).get('path')
```

Comment:
in the yml, the description of `code` is `The base64-encoded contents of the deployment package. Amazon Web Services SDK and CLI clients handle the encoding for you.`
if this is an entry, let's make sure it's clear in the description